There seems to be a problem when the program is terminating if GraphicsMagick has been compiled
with OpenMP support and has used more than one thread. It often segfaults in that case.

See https://bugs.php.net/bug.php?id=78465 and https://gcc.gnu.org/bugzilla/show_bug.cgi?id=91256
--- gmagick.c
+++ gmagick.c
@@ -1709,6 +1709,9 @@
 		return FAILURE;
 	
 	InitializeMagick(cwd);
+#if PHP_VERSION_ID >= 70400
+	SetMagickResourceLimit(ThreadsResource, 1);
+#endif
 	efree(cwd);
 
 	/* init constants */
